README.md

```diff
@@ -1,8 +1,31 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Maths Magician
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+
+## Built With
+
+- React
+- Redux
+- CSS
+- Webpack
+- Html
+
+
+## Live Demo 
+
+[Maths Magician Link](https://fourteen98.github.io/maths-magician/)
```

Comment:
- [ ] OPTIONAL
Kindly note that the live demo link should be the link to your deployed project. Please ensure to remove his redundant link so as to keep your ReadMe professional